presto-main/src/main/java/io/prestosql/security/ViewAccessControl.java

```diff
@@ -74,4 +78,14 @@ public void checkCanGrantExecuteFunctionPrivilege(SecurityContext context, Strin
     {
         return delegate.getColumnMasks(context, tableName, columnName, type);
     }
+
+    private void executeAsViewOwner(Consumer<AccessControl> consumer)
+    {
+        try {
+            consumer.accept(delegate);
+        }
+        catch (AccessDeniedException e) {
+            throw new PrestoException(PERMISSION_DENIED, "Access Denied: View owner does not have sufficient privileges: " + e.getMessage(), e);
```

Comment:
This will result in a repeated "Access Denied" in the message:

> Access Denied: View owner does not have sufficient privileges: Access Denied: View owner 'xxx' cannot create view that selects from yyy

We should strip that off, since we know the original message always has it.

Reply:
Done

Reply:
does the exception message contain "Access Denied" i once now?

Reply:
Yes.